### PR TITLE
[1LP][RFR] Fix test: TestServiceOrderCart

### PR DESCRIPTION
**Pull request by valaparthvi**
## Purpose or Intent

- __Fixing__ 
    1. `TestServiceOrderCart`

### PRT Run
{{ pytest:  -svvv --long-running cfme/tests/services/test_rest_services.py::TestServiceOrderCart --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 2bd7beb009b6af3dc66ccee374ebd23e110e6515

The *local* fixture **`service_templates_class`** is used in the following files:
- **cfme/tests/services/test_rest_services.py**
    - *test_add_to_cart*
    - *test_delete_requests*
    - *test_remove_from_cart*
    - *test_clear_cart*
    - *test_order_cart*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 8d8def20ed3fd207d27bd5414a82e7fcea4b84bd

The *local* fixture **`service_templates_class`** is used in the following files:
- **cfme/tests/services/test_rest_services.py**
    - *test_add_to_cart*
    - *test_delete_requests*
    - *test_remove_from_cart*
    - *test_clear_cart*
    - *test_order_cart*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

File: cfme/tests/services/test_rest_services.py [hunk 1]
```diff
@@ -1914,12 +1920,14 @@ def test_create_second_cart(self, request, appliance, cart):
...
-        request.addfinalizer(second_cart.action.delete)
-        assert second_cart.state == 'cart'
+        with pytest.raises(
+            APIException,
+            match=(
+                "Could not create service order - Validation failed:"
+                " ServiceOrder: State has already been taken"
+            ),
+        ):
+            appliance.rest_api.collections.service_orders.action.create(name="cart2")
```
Comment (ganeshhubale) [inline cfme/tests/services/test_rest_services.py:1931]: Optional:
We can use faux factory.

Comment (valaparthvi) [reply cfme/tests/services/test_rest_services.py:1931]: But the cart will anyway not be created, so keeping it as is.

**Comment by mshriver** [comment]
PRT skipped because of missing `--long-running`, updated PR description to include direct class reference and --long-running

**Pull request merged**